src/runtime/substitution.cc

```diff
@@ -591,11 +592,11 @@ void GraphXfer::find_matches(int depth,
   }
 }
 
+template <typename GraphComp>
```

Comment:
```suggestion
template <typename GraphComparator>
```
"Comp" could also stand for "composition"

Reply:
Good point.